torch/_dynamo/variables/builtin.py

```diff
@@ -326,7 +326,10 @@ def call_function(
             try:
                 inspect.signature(handler).bind(tx, *args, **kwargs)
             except TypeError as exc:
-                log.warning(f"incorrect arg count {handler} {exc}")
+                if not has_constant_handler:
```

Comment:
seems if `if has_constant_handler:` branch is executed before `if handler`, then this warning for dict call can be avoided.

Reply:
Actually no, we get the value of ```has_constant_handler``` at [here](https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/variables/builtin.py#L234), but executing the constant handler at [later](https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/variables/builtin.py#L343).